snapcraft.yaml

```
@@ -915,13 +931,22 @@ parts:
       - lib/*/libnuma*so*
       - lib/*/libpixman*so*
       - lib/*/libusbredir*so*
+      - lib/*/libfdt*.so*
       - share/qemu/keymaps*
```

This is a directory, so I'd be explicit about it, see https://packages.ubuntu.com/jammy/all/qemu-system-data/filelist
```suggestion
      - share/qemu/keymaps/*
```